### ISSUE TITLE
Class drupol\PhpConventions\Grumphp\Task\ComposerNormalize does not exist

### ISSUE THREAD

**Issue by elgandoz** [drupol/drupal-conventions#6]
## Steps required to reproduce the problem
1. `composer create-project drupal-composer/drupal-project:8.x-dev testground --no-interaction`
2. `cd testground/ && composer require --dev drupol/drupal-conventions`
3. `vi composer.json`, adding extra grumphp config
2. `git init && git add . && git commit -m "first commit"`

## Expected Result

* Committed files being checked by Grumphp

## Actual Result

* Error on commit (x2):
```bash
In ContainerBuilder.php line 1175:

  Class drupol\PhpConventions\Grumphp\Task\ComposerNormalize does not exist
```
The commit successfully go through, but no check performed.

**Comment by elgandoz** [comment]
This issue first happened to me while working on an existing project, with drupal-convention successfully working.
In a big merge with several conflicts, I opted to remove `vendor/` and `composer.lock` and install everything again: that's when I experienced the issue. 
Later I built a vanilla drupal installation few times, with the same issue.

Commenting `services.task.composer_normalize`in the referenced grumphp.yml allows the tasks to be performed (and apparently working, given it detects wrong indentation in the project's  composer.json and any other coding standard issues)
